org.osgi.test.junit5/src/test/java/org/osgi/test/junit5/bundle/BundleArgumentProviderTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) OSGi Alliance (2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.osgi.test.junit5.bundle;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.osgi.framework.Bundle;
+import org.osgi.test.assertj.bundle.BundleAssert;
+import org.osgi.test.assertj.dictionary.DictionaryAssert;
+
+public class BundleArgumentProviderTest {
```

Comment:
Since the thing being tested is `BundleArgumentsProvider`, I think this should use the `s`: `BundleArgumentsProviderTest`

Reply:
s!